crun.1.md

```diff
@@ -294,6 +294,19 @@ Path to the file containing the resources to update.
 
 # Extensions to OCI
 
+## `run.oci.seccomp.receiver=PATH`
+
+If the annotation `run.oci.seccomp.receiver=PATH` is specified, the
```

Comment:
When the path is relative, what is the base dir of the path?

Reply:
good catch.  I've changed to be an absolute path